Cannot run examples/hello out of the box

Essentially I did
```zsh
git clone git@github.com:ServiceWeaver/weaver.git
cd weaver/examples/hello
go mod init hello
go mod tidy
weaver generate .
go run .
```
and got output
```
hello listener available on 127.0.0.1:12345
panic: http: multiple registrations for /healthz
```
followed by a stack trace pointing [here](https://github.com/ServiceWeaver/weaver/blob/main/examples/hello/main.go#L57).

Have I missed a step? Is it not expected that I could run the example like that? Where is that prior call to register `/healthz` happening?

--- a/examples/hello/main.go
+++ b/examples/hello/main.go
@@ -52,11 +52,5 @@ func main() {
 			}
 			fmt.Fprintf(w, "Hello, %s!\n", reversed)
 		}))
-
-	// Serve the /healthz endpoint.
-	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "OK")
-	})
-
 	http.Serve(lis, nil)
 }

